apis/management.cattle.io/v3/backup_types.go

```diff
@@ -0,0 +1,63 @@
+package v3
+
+import (
+	"github.com/rancher/norman/condition"
+	"github.com/rancher/norman/types"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	BackupConditionCreated   condition.Cond = "Created"
+	BackupConditionCompleted condition.Cond = "Completed"
+)
+
+type BackupTarget struct {
+	// s3 target
+	S3BackupTarget *S3BackupTarget `yaml:",omitempty" json:"s3BackupTarget"`
+}
+
+type S3BackupTarget struct {
+	// Access key ID
+	AccessKeyID string `yaml:"access_key_id" json:"accessKeyId,omitempty"`
+	// Secret access key
+	SecretAccesssKey string `yaml:"secret_access_key" json:"secretAccessKey,omitempty" norman:"required,type=password" `
```

Comment:
JSON/YAML: We call these `accessKey` and `secretKey` (`access_key` / `secret_key`) in other places of the API (EC2 and EKS)